docs/converting.asciidoc

```diff
@@ -12,41 +12,41 @@ Before you start a conversion, be sure that you understand the basics below.
 
 [float]
 [[core-or-ext]]
-=== Core and extended fields
+==== Core and extended fields
 
-* *Core fields.* Fields that are most common across all use cases are called *core fields*. 
+* *Core fields.* Fields that are most common across all use cases are called *core fields*.
 +
 These generalized fields are used by analysis content
 (searches, visualizations, dashboards, alerts, machine learning jobs, reports)
 across use cases. Analysis content designed to operate on these
-fields should work properly on data from any relevant source. 
+fields should work properly on data from any relevant source.
 +
-Focus on populating these fields first. 
+Focus on populating these fields first.
 
-* *Extended fields.* Any field that is not a core field is called an *extended field*. 
+* *Extended fields.* Any field that is not a core field is called an *extended field*.
 Extended fields may apply to more narrow use cases, or may be more open
 to interpretation depending on the use case. Extended fields are more likely to
 change over time.
 
-Each {ecs} <<ecs-fields,field>> in a table is identified as core or extended.
+Each {ecs} <<ecs-field-reference,field>> in a table is identified as core or extended.
```

Comment:
add "in the Level column" at the end of this sentence.

Reply:
I would try to avoid talking about table structure too specifically in other documentation pages. I wouldn't want this to become obsolete (and we forget to update) if we adjust the table structure.

Reply:
++ the key is to mention that these are called levels.  which you've done.